spec/requests/institution_visibility_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Insitution visiblity work access', type: :request, clean: true, multitenant: true do
+  let(:account) { create(:account) }
+  let(:account2) { create(:account) }
+  let(:tenant_user) { create(:user) }
+  let(:tenant2_user) { create(:user) }
+  let(:work) { create(:work, visibility: 'authenticated') }
+
+  before do
+    WebMock.disable!
+    Apartment::Tenant.create(account.tenant)
+    Apartment::Tenant.switch(account.tenant) do
+      Site.update(account: account)
+      tenant_user
+      work
+    end
+    Apartment::Tenant.create(account2.tenant)
+    Apartment::Tenant.switch(account2.tenant) do
+      Site.update(account: account2)
+      tenant2_user
+    end
+  end
+
+  after do
+    WebMock.enable!
+    Apartment::Tenant.drop(account.tenant)
```

Comment:
is there a reason to drop tenant 1 but not tenant 2?

Reply:
Good catch.  Tenant 2 should be dropped here.